client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -63,6 +66,24 @@ var CheckoutThankYou = React.createClass( {
 	},
 
 	componentDidMount: function() {
+		var lastTransaction = this.props.lastTransaction,
+			selectedSite;
+
+		if ( lastTransaction ) {
```

Comment:
True, I added it because I encountered an undefined `lastTransaction` error here (when reloading or going back to this page), but I think it will still throw an error because of those couple of other places.